### DISCUSSION TITLE
123云盘无法挂载，显示：当前账号存在安全风险，请使用短信验证码或者微信进行登录。

### DISCUSSION THREAD

**Discussion by zzw8** [AlistGo/alist#6254] [Q&A]
![mmexport1711121849276.png](https://github.com/alist-org/alist/assets/154610227/58594806-e272-4039-a868-26dd3c2ecfa3)

**Comment by ririye** [comment]
电脑打开网页版123云盘，用手机的微信扫码登录下就好啦

**Reply by jackchen75** [reply]
不行哦，好象非会员不能挂载webdav了

**Reply by ririye** [reply]
我也不是会员，我这里正常的啊
![QQ截图20240329131216](https://github.com/alist-org/alist/assets/124979523/fbbd666b-f79c-44b8-98d4-64066c6ba06b)
![QQ截图20240329131233](https://github.com/alist-org/alist/assets/124979523/70a9f6e3-e4ed-4536-8b53-e33f5c4cab96)
![QQ图片20240329131308](https://github.com/alist-org/alist/assets/124979523/e38b99ce-cfb8-4867-b6e7-91585028431e)

**Reply by katorly** [reply]
你是不是以前挂载的

**Reply by ririye** [reply]
就你提问的当天，我也是同样的问题就在你提问的前几分钟才挂上的

**Reply by ririye** [reply]
你楼下的截图应该是应用在这里的的吧？alist不受限制吧？
![QQ截图20240329133031](https://github.com/alist-org/alist/assets/124979523/55998d4f-d709-4b4b-bef9-2379f71fe165)

**Comment by katorly** [comment]
没机会了，必须开会员  

![image](https://github.com/alist-org/alist/assets/70022443/3b63ff03-f55b-4c94-aa0e-648a019ee091)

**Comment by AspirantH** [comment]
早就料到，这家网盘的政策动不动就改，幸亏没做主力
